client.go

```diff
@@ -1104,6 +1104,14 @@ func (c *Client) handlePongMessage(msg PongMessage) {
 	}
 }
 
+func (c *Client) RestrictChatForFollowers(channel string, duration string) {
+	c.Say(channel, "/followers "+duration)
+}
+
+func (c *Client) AllowChatForEverybody(channel string) {
```

Comment:
Should be called FollowersOff for less confusion

Reply:
okay